benchmarks/performance-poetry/complex-poetry/src/main/java/com/squareup/benchmarks/performance/complex/poetry/MaybeLoadingGatekeeperWorkflow.kt

```diff
@@ -48,5 +49,36 @@ class MaybeLoadingGatekeeperWorkflow<T : Any>(
     )
   }
 
+  @Composable
+  override fun Rendering(
+    renderProps: Unit,
+    renderState: IsLoading,
+    context: RenderContext,
+    hoistRendering: @Composable (rendering: MayBeLoadingScreen) -> Unit
+  ) {
```

Comment:
I'm stumped here. These overrides are recognized by the compiler as it runs in the IDE but when tracing through the call always goes through to the base `StatefulWorkflow::Rendering` implementation?

@zach-klippenstein any idea why this override wouldn't be recognized at runtime?

Reply:
Reproduced in small project. I believe bug in Molecule - https://github.com/cashapp/molecule/issues/91